### ISSUE TITLE
version 3.x?

### ISSUE THREAD

**Issue by ousia** [fedora-haskell/pandoc#15]
@juhp,

I wonder whether there would be a chance to get latest pandoc in copr.

Many thanks for your work and your help.

**Comment by juhp** [comment]
Yes, sure, but it will take a little while - I am busy with wrapping up F38 packages.

Also note that with pandoc-3 the cli executable is now provided by pandoc-cli.

PR also welcome. :)

**Comment by ousia** [comment]
I‘m really sorry, @juhp, but I really don‘t know how to modify the spec file for the newest version of pandoc.

**Comment by juhp** [comment]
That's okay - I will try to get to it sooner or later
